base_custom_attribute/models/attribute_attribute.py

```diff
@@ -127,13 +162,60 @@ def _build_attribute_view(self):
                 groups.append(att_group)
 
             setup_modifiers(subgroup)
-            attribute._build_attribute_field(subgroup)
-        return main_group
+            attribute.with_context(
+                {"include_native_attribute": context_include}
+            )._build_attribute_field(subgroup, native_efields)
+
+        return main_att_group
+
+    def _insert_attribute(self, eview):
+        """Insert all the model's Attributes related fields into the eview at
+        the placeholder's place.
+        This method is overriden in the "attribute.set.owner.mixin" in order to be used
+        by the corresponding model using Attributes."""
+        context_include = self._context.get("include_native_attribute")
+        form_name = eview.get("string")
+        placeholder = eview.xpath("//separator[@name='attributes_placeholder']")
+
+        if len(placeholder) != 1:
+            raise ValidationError(
+                _(
+                    """It is impossible to add Attributes on "{}" xml view as there is
+                    not one "<separator name="attributes_placeholder" />" in it.
+                    """.format(
+                        form_name
+                    )
+                )
+            )
+        # Remove attributes related to native fields
+        native_efields = self._remove_native_fields(eview) if context_include else []
+        # Build the Attributes group element
+        main_att_group = self.with_context(
+            {"include_native_attribute": context_include}
+        )._build_attribute_view(native_efields)
+
+        placeholder[0].getparent().replace(placeholder[0], main_att_group)
+        return eview
 
     field_id = fields.Many2one(
         "ir.model.fields", "Ir Model Fields", required=True, ondelete="cascade"
     )
 
+    # 'attribute_nature' is only an interface field, use 'is_custom' in business logic
+    attribute_nature = fields.Selection(
+        [("custom", "Custom"), ("native", "Native")],
+        string="Attribute Nature",
+        compute="_compute_attribute_nature",
+        inverse="_inverse_attribute_nature",
+        default="custom",
+    )
+    is_custom = fields.Boolean(
```

Comment:
Do not add a "is_custom".
We can simply in the python code do a attribute_nature == 'custom'.
Less code is always better